resources/js/visual-editor/site-editor/styles/styles-inspector.css

```
@@ -0,0 +1,196 @@
+    min-width: 0;
+    padding: 0.5rem 0.75rem !important;
+    text-align: left;
+    width: 100%;
+}
+
+.ap-site-editor__style-listing-label {
+    flex: 1;
+    font-weight: 500;
+    min-width: 0;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)